lib/catalog/evaluate_order_process.rb

```diff
@@ -52,7 +52,7 @@ def find_relevant_order_processes
     end
 
     def all_tags
-      tag_pattern = '\/\w+\/order_processes=\d'
+      tag_pattern = '\/\w+.\w+\/order_processes=\d'
```

Comment:
@hsong-rh What happened here why would it be different?

Reply:
To support app name like `catalog-inventory`